### DISCUSSION TITLE
can I create an alias with N email addresses

### DISCUSSION THREAD

**Discussion by ivanbishop** [the-djmaze/snappymail#1239] [Q&A]
Hi, I wondered is there any support top create an alias, like

list1@server.com 
add 10 user addresses to list1

THEN

via browser send mail to list1 and have 10 people mailed?

?

Manipulate SQLLITE db?

Thanks
Ivan

**Comment by the-djmaze** [comment]
Not at the moment.
There's a feature request issue open for contact groups.
